src/components/Section/Package/Package.jsx

```diff
@@ -82,28 +81,40 @@ class Package extends SectionElement {
    * releases or errors based on whether the form is valid
    */
   onTransitionEnd() {
-    const tally = this.errorCheck()
-    for (const sectionName in tally) {
-      const mark = tally[sectionName]
-      if (mark.errors > 0) {
-        // We do not call `this.props.update` here because there is no hard
-        // route to `package/errors`. By only pushing the new location we keep
-        // the navigation element marking `/review` as active but allow us to
-        // display a different internal route.
-        this.props.history.push('/form/package/errors')
-        return
-      }
+    const formIsValid = this.isValid()
+
+    if (!formIsValid) {
+      // We do not call `this.props.update` here because there is no hard
+      // route to `package/errors`. By only pushing the new location we keep
+      // the navigation element marking `/review` as active but allow us to
+      // display a different internal route.
+      this.props.history.push('/form/package/errors')
+      return
     }
 
     this.props.update({ section: 'package', subsection: 'submit' })
     return
   }
 
-  /**
-   * TODO: Remove after testing. Hook to get to releases form
-   */
-  goToReleases() {
-    this.props.update({ section: 'package', subsection: 'submit' })
+  isValid(params = null) {
+    const tally = this.errorCheck()
+
+    // Allows invalid form submission for debugging purpose
+    if (params) {
+      params = queryString.parse(params)
+
+      if (env.IsDevelopment() && params.force === 'true') {
+        return true
+      }
+    }
+
+    for (const sectionName in tally) {
+      const mark = tally[sectionName]
+      if (mark.errors > 0) {
+        return false
+      }
+      return true
```

Comment:
I think this will return true the first time a `mark` doesn't have any errors, is that the desired behavior?

Reply:
Nope. Updated https://github.com/18F/e-QIP-prototype/pull/933/commits/27173e6961d903fc24b91df035fe61975a789a98